source/agora/test/Base.d

```diff
@@ -2141,7 +2141,7 @@ public APIManager makeTestNetwork (APIManager : TestAPIManager = TestAPIManager)
             registry_address : "http://name.registry",
             recurring_enrollment : test_conf.recurring_enrollment,
             name_registration_interval : 10.seconds,
-            preimage_reveal_interval : 1.seconds,  // check revealing frequently
+            preimage_reveal_interval : 500.msecs,  // check revealing frequently
```

Comment:
Do we really need to decrease the interval of revealing of pre-images because it reveals the pre-images 6 height after? I don't think this is necessary. But the determination is yours.

Reply:
I am not sure this changed anything so will revert.